packages/manager/modules/nasha/src/dashboard/partition/accesses/delete/index.js

```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerNashaDashboardPartitionAccessesDeleteLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('nasha.dashboard.partition.accesses.delete.**', {
+      url: '/:customSnapshotName/delete',
```

Comment:
param names are different between lazyload and state declaration. We should keep the consistency

Reply:
Good catch :+1: